subsys/bluetooth/host/classic/avrcp.c

```diff
@@ -129,7 +174,11 @@ static struct bt_sdp_attribute avrcp_tg_attrs[] = {
 		},
 		)
 	),
-	BT_SDP_SUPPORTED_FEATURES(AVRCP_CAT_1 | AVRCP_CAT_2),
+	BT_SDP_SUPPORTED_FEATURES(AVRCP_CAT_1 | AVRCP_CAT_2
+#if defined(CONFIG_BT_AVRCP_BROWSING)
+				 |AVRCP_BROWSING_SUPPORT
+#endif /* CONFIG_BT_AVRCP_BROWSING */
+	),
```

Comment:
My suggestion is,
Define a macro AVRCP_BROWSING_ENABLE,
```
#if defined(CONFIG_BT_AVRCP_BROWSING)
#define AVRCP_BROWSING_ENABLE AVRCP_BROWSING_SUPPORT
#else
#define AVRCP_BROWSING_ENABLE 0
#endif /* CONFIG_BT_AVRCP_BROWSING */
```

```suggestion
	BT_SDP_SUPPORTED_FEATURES(AVRCP_CAT_1 | AVRCP_CAT_2 | AVRCP_BROWSING_ENABLE),
```